includes/view/admin-page-view.php

```diff
@@ -0,0 +1,67 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! current_user_can( 'manage_options' ) ) {
+	wp_die( 'Unauthorized user' );
```

Comment:
we might want to add i18n function here, not critical, as this is a generic error